Package.resolved

```diff
@@ -9,22 +9,31 @@
         "version" : "6.0.1"
       }
     },
+    {
+      "identity" : "jinja",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/maiqingqiang/Jinja",
+      "state" : {
+        "branch" : "main",
+        "revision" : "5b0703d19a8901b76948753e5c5e3ca77043d33f"
```

Comment:
Can you make a tag for this?  As people use mlx-swift-examples in their own code they had expressed a preference for tags.